src/Network/Network.Test/ScenarioTests/VirtualNetworkTests.ps1

```diff
@@ -1292,3 +1292,56 @@ function Test-VirtualNetworkSubnetServiceEndpointPolicies
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Tests creating new virtual network with flow timeout.
+#>
+function Test-VirtualNetworkCRUD-FlowTimeout
+{
+    # Setup
+    $rgname = Get-ResourceGroupName
+    $rname = Get-ResourceName
+    $rglocation = Get-ProviderLocation ResourceManagement
+    $resourceTypeParent = "Microsoft.Network/virtualNetworks"
+    $location = Get-ProviderLocation $resourceTypeParent
+
+    try
+    {
+        # Create the resource group
+        $resourceGroup = New-AzResourceGroup -Name $rgname -Location $rglocation -Tags @{ testtag = "testval" }
+
+        # Create virtual network
+        $actual = New-AzVirtualNetwork -ResourceGroupName $rgname -name $rname -location $location -FlowTimeout 15 -AddressPrefix 10.0.0.0/16
+        $expected = Get-AzVirtualNetwork -ResourceGroupName $rgname -name $rname
+        Assert-AreEqual $expected.ResourceGroupName $actual.ResourceGroupName 
+        Assert-AreEqual $expected.Name $actual.Name 
+        Assert-AreEqual $expected.Location $actual.Location
+        Assert-NotNull $expected.ResourceGuid
+        Assert-AreEqual "Succeeded" $expected.ProvisioningState
+        Assert-AreEqual 15 $expected.FlowTimeoutInMinutes
```

Comment:
Hi @Satya-anshu , our pipeline said 'Assertion failed because expected '15' does not match actual '. Could you figure it out?